### DISCUSSION TITLE
Add RTC setting

### DISCUSSION THREAD

**Discussion by Real96** [rafaelvcaetano/melonDS-android#395] [Ideas]
Can RTC option be added under System setting? This way, games can be started at a specified Date and Hour:Minute:Second

**Comment by rafaelvcaetano** [comment]
This needs to be implemented by the PC version. They already have a PR open for that feature (https://github.com/Arisotura/melonDS/pull/973), but it hasn't been accepted yet.

**Reply by Real96** [reply]
I didn't know. I hope it will be done sooner or later
